contrib/msggen/msggen/__main__.py

```diff
@@ -1,153 +1,31 @@
-from msggen.model import Method, CompositeField, Service
```

Comment:
I think there should be a document on how to use this package. Or at least this file is a example of how to use.

Reply:
Good idea! I will add an example dir!